app/components/Modals/ValuesMessageModal/ValuesMessageModal.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import useUser from 'app/utils/hooks/useUser';
+import * as styled from 'app/components/Modals/ValuesMessageModal/ValuesMessageModal.Styled';
+import Button from 'app/components/Atoms/Button';
+import { BsCircleFill } from 'react-icons/bs';
+import { SECONDARY_BUTTON } from 'app/utils/constants';
+
+function ValuesMessageModal({ show, onClose }) {
+  const profile = useUser();
+
+  const renderBulletPoint = (color) => (
+    <BsCircleFill color={color} size="7px" style={{ marginTop: '3px', marginRight: '10px' }} />
+  );
+
+  if (!show) { return null; }
+  return (
+    <div onClick={onClose}>
+      <styled.Modal onClicl={onClose}>
```

Comment:
I think your have a little typo here.
```suggestion
      <styled.Modal onClick={onClose}>
```

Reply:
Done